Revit_Engine/Modify/SetPullEdges.cs

```diff
@@ -51,6 +51,11 @@ public static FilterRequest SetPullEdges(this FilterRequest filterRequest, bool
             return aFilterRequest;
         }
 
+
+        /***************************************************/
+        /**** Deprecated Methods                        ****/
+        /***************************************************/
+
         [Deprecated("2.2")]
```

Comment:
Can we delete any deprecated method that is prior to 2.4 and only leave in the ones deprecated from 3.0 onwards?

Reply:
I would rather resolve this with another issue + PR, no to add too much stuff to this one.